.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -916,6 +916,12 @@ if [ "${WORKER}" == 0 ]; then
     fi
 fi
 
+# Upgrade to http/2 with TLS for non-worker Vagrant VMs
```

Comment:
`install_submitty` could be run multiple times in the same VM, especially frequently for developers working outside the site development code. Is it safe to run the dev-upgrade scripts multiple times with it? Would it make sense to move this to `install_system` if possible? So it will only run with the initial vagrant up

Reply:
Thank you for reviewing.  Most of the codes are safe to rerun -- it will remove the changes then re-applying -- except that the certificate might expire in 730 days since the script launched.  I just added an expiration check.